paper.md

```diff
@@ -1,32 +1,109 @@
 ---
-title: 'sourmash: a library for MinHash sketching of DNA'
+title: 'sourmash: a tool to quickly search, compare, and analyze genomic and metagenomic data sets'
```

Comment:
"genomic and metegenomic" ignores transcriptome/metatranscriptome data, etc. But since we have mostly tested/developed workflows for genomic/metagenomic data, this is probably fine?

Reply:
yeah, that was basically my thinking here :)